server-helpers.js

```diff
@@ -37,14 +37,14 @@ class UserHelper {
 
   static checkLoginInfo = (info) => {
     const infoProvided = Object.keys(info)
-    const result = { hasEnoughInfo: false, message: '', userType: ''}
+    const result = { hasEnoughInfo: false, message: '', username: '' }
     if (!infoProvided.includes("password")) {
       result.message = 'You need to provide a password'
-    } else if (!infoProvided.some(info => ["email", "name"].includes(info))) {
+    } else if (infoProvided.length === 0 && infoProvided[0] === 'password') {
```

Comment:
Is it save to assume the `username`? will always be the first info? What if there's extra info?

Reply:
I'm a little confused by this line, if `infoProvided.length === 0` how can there be an `infoProvided[0]`?